bot/kodiak/queue.py

```diff
@@ -143,7 +143,7 @@ async def requeue() -> None:
             only_if_not_exists=True,
         )
 
-    async def queue_for_merge() -> Optional[int]:
+    async def queue_for_merge(*, first: bool) -> Optional[int]:
```

Comment:
why does this function exist?

Reply:
Because `evaluate_pr` requires it, but when we're merging a PR we will never call it.